ui/pages/confirm-transaction-base/transaction-alerts/transaction-alerts.js

```diff
@@ -151,7 +151,7 @@ const TransactionAlerts = ({
           message={
             <Typography
               align="left"
-              margin={[0, 0]}
+              margin={0}
               tag={TYPOGRAPHY.Paragraph}
               variant={TYPOGRAPHY.H7}
             >
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props